CHANGELOG.md

```diff
@@ -136,7 +136,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
 - Renamed the `TrainingTypePlugin` to `Strategy` ([#11120](https://github.com/PyTorchLightning/pytorch-lightning/pull/11120))
-
+    * Renamed `ParallelPlugin` to `ParallelStrategy` ([#11123](https://github.com/PyTorchLightning/pytorch-lightning/pull/11123))
```

Comment:
```suggestion
- Renamed `ParallelPlugin` to `ParallelStrategy` ([#11123](https://github.com/PyTorchLightning/pytorch-lightning/pull/11123))
```

Reply:
Let's group all Accelerator Refactor Entries together in the CHANGELOG, this would get confusing for the readers.

Reply:
You are suggesting not to group them?

Reply:
Grouped them together now. Some PRs have been merged out of order